ansible/roles/dashmate/tasks/main.yml

```diff
@@ -173,8 +173,24 @@
   changed_when: true
   when: dashmate_branch is defined
 
-# TODO: we should use status command to figure out if dashmate is running
-#   to get the status from the response
+- name: Get status
+  ansible.builtin.command: "{{ dashmate_cmd }} status --format=json"
+  become: true
+  become_user: dashmate
+  changed_when: true
+  args:
+    chdir: '{{ dashmate_cwd }}'
+  register: dashmate_status
+
+- name: Get service status
```

Comment:
it might be situation that core is started but platform is not (after platform destroy) so you need to check both, drive and core. If core is started but not a platform you should use `start --platform`